src/log.py

```diff
@@ -0,0 +1,94 @@
+"""
+log
+
+By: Norma Chang
+"""
+import json
+import time
+import os.path
+
+
+class Log:
+    """
+    Logging class.
+
+    Logs data from the properties of an instance of an vehicle class for
```

Comment:
1. typo (an vehicle)
2. It's not just for testing purposes. It's for general viewing as well. 
3. You can just say "logs data from a vehicle."

Reply:
changed to: Logs data from a vehicle.